ha_lineairdb.cc

```diff
@@ -255,6 +255,15 @@ ha_lineairdb::ha_lineairdb(handlerton* hton, TABLE_SHARE* table_arg)
       current_position_(0),
       blobroot(csv_key_memory_blobroot, BLOB_MEMROOT_ALLOC_SIZE) {}
 
+void ha_lineairdb::set_key_and_key_part_info(const TABLE* const table) {
+  key_info = table->key_info;
+  primary_key_type = (ha_base_keytype)table->key_info[key_info_pk_index].key_part[0].type;
+
+  key_part = table->key_info->key_part;
+  indexed_key_part = key_part[0];
+  num_key_parts = table->s->key_parts;
```

Comment:
[NOTE]
It seems the `KEY_PART* key_part` is the pointer. 
So you may have to consider that this pointer will be freed earlier, when the lifetime of the LineairDB is longer than that of the `Table`. ( Though it is not necessary to think about it now, since delete is not yet supported).

Reply:
LineairDB-storage-engine does not access `key_part` after table is deleted unless `open` is called again.
So I think we do not need to worry about accessing to freed memory in this case, although I did not really consider the risk at the moment of writing this code. I will be more aware.

Reply:
> LineairDB-storage-engine does not access key_part after table is deleted unless open is called again.

Okay. I think so too ... Let us not forget to implement so.